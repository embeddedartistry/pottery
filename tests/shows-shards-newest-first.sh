pottery init
TEST_TIME=1992-01-12T16:34:23Z pottery post "First post"
TEST_TIME=1992-01-12T17:00:00Z pottery post "Second post."
TEST_TIME=1992-01-13T16:34:23Z pottery post "It's a new day."
pottery show
