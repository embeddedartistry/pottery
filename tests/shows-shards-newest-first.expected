pottery init
TEST_TIME=1992-01-12T16:34:23Z pottery post "First post"
docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_post_FAKEUNIQUEID.md
TEST_TIME=1992-01-12T17:00:00Z pottery post "Second post."
docs/project-history/1992/1992-01/1992-01-12T17:00:00Z_post_FAKEUNIQUEID.md
TEST_TIME=1992-01-13T16:34:23Z pottery post "It's a new day."
docs/project-history/1992/1992-01/1992-01-13T16:34:23Z_post_FAKEUNIQUEID.md
pottery show
1992-01-13T16:34:23Z

It's a new day.

1992-01-12T17:00:00Z

Second post.

1992-01-12T16:34:23Z

First post

